fm.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/fm/impl/LoggingFaultReporter.java

```diff
@@ -26,16 +27,23 @@
 public class LoggingFaultReporter implements RepairFaultReporter
 {
     private static final Logger LOG = LoggerFactory.getLogger(LoggingFaultReporter.class);
+    Map<Integer, FaultCode> alarms = new HashMap<>();
 
     @Override
     public void raise(FaultCode faultCode, Map<String, Object> data)
     {
+        alarms.put(data.hashCode(), faultCode);
         LOG.error("Raising alarm: {} - {}", faultCode, data);
     }
 
     @Override
     public void cease(FaultCode faultCode, Map<String, Object> data)
     {
-        LOG.info("Ceasing alarm: {} - {}", faultCode, data);
+        FaultCode code = alarms.get(data.hashCode());
+        if (code != null)
+        {
+            LOG.info("Ceasing alarm: {} - {}", code , data);
```

Comment:
You decided to keep it as info and not debug? I have no objections if that's how you decided.

Reply:
Yes, keep at info, but only log it if it ceasing an alarm that occurred earlier.

Reply:
LGTM